EMS/common-bundle/src/Storage/ArchiveItem.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EMS\CommonBundle\Storage;
+
+class ArchiveItem implements \JsonSerializable
+{
+    public const FILENAME = 'filename';
+    public const HASH = 'hash';
+    public const TYPE = 'type';
+    public const SIZE = 'size';
+
+    public function __construct(
+        private readonly string $filename,
+        private readonly string $type,
+        private readonly int $size,
+        private readonly string $hash
+    ) {
+    }
+
+    public function getFilename(): string
+    {
+        return $this->filename;
+    }
+
+    public function getType(): string
+    {
+        return $this->type;
+    }
+
+    public function getHash(): string
+    {
+        return $this->hash;
+    }
+
+    public function getSize(): int
+    {
+        return $this->size;
+    }
```

Comment:
the properties are readonly but private. The getters are overkill you could just make the properties public and remove the getters. 

```suggestion
    public function __construct(
        public readonly string $filename,
        public readonly string $type,
        public readonly int $size,
        public readonly string $hash
    ) {
    }

```